src/azure-cli-testsdk/azure/cli/testsdk/base.py

```diff
@@ -181,6 +184,12 @@ def __init__(self, cli_ctx, command, expect_failure=False):
         self.applog = ''
         self.command_coverage = {}
 
+        if os.environ.get(ENV_COMMAND_COVERAGE, None) == 'True':
+            with open(COVERAGE_FILE, 'a') as coverage_file:
```

Comment:
`'a'` would continue editing from the end of the file instead of overwriting. Is this intended?

Reply:
If intended, then user has to remember to delete the file later?

Reply:
Yes, and yes.

Reply:
The main reason is that I didn't want the script to have to run all the tests in order to do coverage analysis, because running tests is pretty dang slow.  That's why there is the warning message when you run the script.  I wanted to get this core logic in because I have other things to move onto for this sprint, but I think in the future they test script could optionally be forwarded to this or vice versa.

Reply:
Also, if this moves to be Knack-based I don't want to gold-plate this version. I'd rather gold plate that version. :D